### DISCUSSION TITLE
Notes Section on Recipe Card

### DISCUSSION THREAD

**Discussion by pajades** [nextcloud/cookbook#1930] [Ideas and discussions]
Feature Request: add a '**Notes**' section to the bottom of the recipe card.   

I like to make notes about the date I made a recipe with any additions / substitutions / finished product qualities, etc.

It's also a good place to put extra information about certain ingredients in a recipe that don't belong in the ingredients, instructions or description fields.

**Comment by christianlupus** [comment]
May I ask you to clarify: The notes should be in the details view of a recipe of in the list of all recipes in the cookbook?

**Reply by pajades** [reply]
In the details view of the individual recipe.

**Reply by christianlupus** [reply]
Maybe this is what you are looking for? https://github.com/nextcloud/cookbook/issues/782

**Reply by pajades** [reply]
This is what I'm looking for (see the "Notes" section at the bottom of the recipe image):
![RecipeCageCardCapture](https://github.com/nextcloud/cookbook/assets/60847505/c208ad13-1b66-44b6-a6d1-ba704169b3a6)

**Comment by christianlupus** [comment]
Closing in favor of #782 as the image was posted there as well.
